src/sentry/discover/compare_timeseries.py

```diff
@@ -196,10 +201,10 @@ def get_mismatch_type(mismatches: dict[int, dict[str, float]]):
     if all_rpc_values_zero:
         return MismatchType.RPC_ALWAYS_ZERO, many_low_conf_buckets, many_low_sample_rate_buckets
 
-    if snql_always_lower:
+    if snql_always_lower and total_buckets - len(mismatches) < 2:
```

Comment:
snql can have an extra bucket (because of how we group by timestamps)

Reply:
curious, is `snql` the query layer we use via snuba?

Reply:
@nsdeschenes yes - it stands for Snuba Query Language ([docs](https://getsentry.github.io/snuba/language/snql.html)) it used to be the primary way we (the sentry backend) would query clickhouse via snuba. but with the new RPCs, it's not longer the case (although the platform itself may or may not use SnQL under the hood)